### ISSUE TITLE
Remove one contact information on "edit submisssion page"

### ISSUE THREAD

**Issue by jonquet** [agroportal/agroportal_web_ui#103]
On peut pas supprimer un contact de la page de soumission d'unes avec un bouton e ontologie. 
Si on efface le champ, il revient peuplé avec la même info que précédement. 

On vient de voir ca sur Agrovoc. 

Qu'en est-il des autres propriétés qui sont ajoutées avec un bouton "Add .."

**Comment by vemonet** [comment]
On peut le faire avec REST apparemment en cas de besoin. Anne tu peux confirmer ?

Pour les autres metadata où on peut ajouter des "New value" c'est possible de supprimer des champs en les vidant dans l'UI (testé pour useImports)

**Comment by antool** [comment]
Oui, oui, confirmé : quand on passe un (ou plusieurs) contact avec REST, cela écrase les contacts précédents (quel que soit leur nombre)

**Comment by vemonet** [comment]
Ok, je pense que le mieux pour le moment ça serait de le documenter à un endroit facile à retrouver. Mettre un exemple dans le document AgroPortal administration par exemple : https://docs.google.com/document/d/1ISW9kyETrXxjXFsTB7LE9VQJrgD1SE6dMaJeYMiQpfU/edit#heading=h.i33arpt7p3eq )

Et si c'est vraiment nécessaire on fera des modifications dans l'UI, mais ça semble être une manipulation assez peu courante
